Makefile

```diff
@@ -600,7 +600,7 @@ glance_deploy_prep: glance_deploy_cleanup ## prepares the CR to install the serv
 	mkdir -p ${OPERATOR_BASE_DIR} ${OPERATOR_DIR} ${DEPLOY_DIR}
 	pushd ${OPERATOR_BASE_DIR} && git clone -b ${GLANCE_BRANCH} ${GLANCE_REPO} && popd
 	cp ${GLANCE_CR} ${DEPLOY_DIR}
-	bash scripts/gen-service-kustomize.sh
+	bash scripts/gen-glance-kustomize.sh
```

Comment:
I'm wondering if we can find a way to generalize a single script instead of duplicating it for a 90% of its content.
An idea could be running something like:

```
bash scripts/gen-service-kustomize.sh "glance"
```

and within the script, to avoid messing up w/ a bunch of if statements, you can buld a function that renders what's needed for glance.
I'm telling this because I see a pattern here, and it would be great solving for all the services instead of creating ad-hoc scripts.
I see, on the other hand, that maintaining a script for each service is easier, so it would make sense having a "common" part that you can `source` within the script.

Reply:
I decided to create a service-specific script because that is already done for telemetry. We can create more generic script probably which I'm trying in the latest change.